### PR TITLE
Product Detail Images Component Testing

### PR DESCRIPTION
**Pull request by saifulislamdev**

### CODE REVIEW

File: src/components/Product/Specifications.test.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
+import React from 'react';
```
Comment (saifulislamdev) [inline src/components/Product/Specifications.test.js:1]: There is no real change in this file besides lines 6 and 126.

File: src/components/Product/ProductDetailImages.test.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+describe('ProductDetailImages', () => {
+    it('Loads with all images', () => {
+        const { getByLabelText } = render(
+            <ProductDetailImages images={images} maxQuantity={123} />
+        );
+        const imagesInDOM = document.getElementsByTagName('img');
+        expect(imagesInDOM).toHaveLength(images.length + 1); // loads all images
+        [...imagesInDOM].forEach((image, i) => {
+            if (i !== 0) expect(image.src).toContain(images[i - 1].src);
+            else expect(image.src).toContain(images[0].src);
```
Comment (zealptl) [inline src/components/Product/ProductDetailImages.test.js:30]: should this be images[0] or images[i]?

File: src/components/Product/ProductDetailImages.test.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        const nextImgButton = getByLabelText('Look at next image');
+        expect(nextImgButton).toBeDisabled();
+    });
+
+    it('Has the correct alt text for each image', () => {
+        render(<ProductDetailImages images={images} maxQuantity={123} />);
+        const imagesInDOM = document.getElementsByTagName('img');
+        [...imagesInDOM].forEach((image, i) => {
+            if (i !== 0) expect(image.alt).toContain(images[i - 1].name);
+            else expect(image.alt).toContain(images[0].name);
```
Comment (zealptl) [inline src/components/Product/ProductDetailImages.test.js:56]: same thing here?

File: src/components/Product/ProductDetailImages.test.js (hunk 2 above)
Comment (saifulislamdev) [reply src/components/Product/ProductDetailImages.test.js:30]: Both works, but `images[0]` gives it more specificity, and readers will avoid an extra step in trying to understand what's going on here.

File: src/components/Product/ProductDetailImages.test.js (hunk 3 above)
Comment (saifulislamdev) [reply src/components/Product/ProductDetailImages.test.js:56]: Same answer :)

**Review by zealptl — APPROVED** [review]

**Pull request merged**